src/src/FileItem.java

```diff
@@ -157,4 +163,41 @@ private static String getFileExtension(File file)
         
         return extensionFile;
     }  
+    
+    /**
+     * Method incharge  to  run command  to  update value  for  attribute in filesystem
+     * @param fileLocation String  that cotains the absolut  path of  file
+     * @param attributeName String that contains name of  attribute
+     * @param attributeValue String  thta  contains de value  of  attribute
+     * @return boolean true if command run sucees  
+     */
+    private boolean runChange(String fileLocation, String attributeName, String attributeValue )
+    {
+        String commandRun = "";
+        
+        if(attributeName.equals("ReadOnly"))
+        {
+            if(attributeValue.equals("Enabled"))
+            {
+                commandRun = RunCommand.runCommandWindows("attrib +r "+ fileLocation );
```

Comment:
I will suggest in this part to build an string like "attrib +r +h" and just run the command just one time